assignments/prototypes.js

```diff
@@ -92,19 +127,124 @@
     language: 'Elvish',
   });
 
-  console.log(mage.createdAt); // Today's date
-  console.log(archer.dimensions); // { length: 1, width: 2, height: 4 }
-  console.log(swordsman.healthPoints); // 15
-  console.log(mage.name); // Bruce
-  console.log(swordsman.team); // The Round Table
-  console.log(mage.weapons); // Staff of Shamalama
-  console.log(archer.language); // Elvish
-  console.log(archer.greet()); // Lilith offers a greeting in Elvish.
-  console.log(mage.takeDamage()); // Bruce took damage.
-  console.log(swordsman.destroy()); // Sir Mustachio was removed from the game.
-*/
+  // console.log(mage.createdAt); // Today's date
+  // console.log(archer.dimensions); // { length: 1, width: 2, height: 4 }
+  // console.log(swordsman.healthPoints); // 15
+  // console.log(mage.name); // Bruce
+  // console.log(swordsman.team); // The Round Table
+  // console.log(mage.weapons); // Staff of Shamalama
+  // console.log(archer.language); // Elvish
+  // console.log(archer.greet()); // Lilith offers a greeting in Elvish.
+  // console.log(mage.takeDamage()); // Bruce took damage.
+  // console.log(swordsman.destroy()); // Sir Mustachio was removed from the game.
 
   // Stretch task: 
   // * Create Villain and Hero constructor functions that inherit from the Humanoid constructor function.  
   // * Give the Hero and Villains different methods that could be used to remove health points from objects which could result in destruction if health gets to 0 or drops below 0;
-  // * Create two new objects, one a villain and one a hero and fight it out with methods!
\ No newline at end of file
+  // * Create two new objects, one a villain and one a hero and fight it out with methods!
+
+  function HeroVillan(attributes) {
+    Humanoid.call(this, attributes);
+      this.ability = attributes.ability;
+      this.finish = attributes.finish;
+  }
+  
+  HeroVillan.prototype = Object.create(Humanoid.prototype);
+  
+  HeroVillan.prototype.fight1 = function (target) {
+    if (target.healthPoints <= 0){
+    console.log(target.name + ' is looking hurt. Finish Him!')
+    return `${this.name} finishes ` + target.name + ` with ${this.finish}!!!`
+    }
+    else {
+    target["healthPoints"] = (target["healthPoints"] - 10)
+    return `${this.name} lashes out with the ${this.ability} ability.`;
+    }
+  };
+
+  HeroVillan.prototype.fight2 = function (target) {
+    if (target.healthPoints <= 0){
+    console.log(target.name + ' is looking hurt. Finish Him!')
+    return `${this.name} finishes ` + target.name + ` with ${this.finish}!!!`;
+    }
+    else {
+    target["healthPoints"] = (target["healthPoints"] - 15)
```

Comment:
I've noticed that you're using the bracket notation a lot.  It's perfectly fine, but I've found that it's much more common to use the dot notation.

Reply:
Changed in the JSIV project to reflect dot notation.